src/bufr/IodaEncoder/IodaDescription.h

```diff
@@ -57,6 +104,9 @@ namespace Ingester
         /// \brief Add Variable defenition
         void addVariable(const VariableDescription& variable);
 
+        /// \brief Add Globals defenition
+//       void addGlobal(const GlobalDescription& global);
```

Comment:
You are now ready to implement this method. You can pass it a `const std::shared_ptr<GlobalDescriptionBase>&` and go from there.